pkg/store/vc/store.go

```diff
@@ -59,6 +64,12 @@ func (c *callback) storeVC(this js.Value, inputs []js.Value) interface{} {
 
 		if err := c.store.SaveVC(vc); err != nil {
 			m["data"] = fmt.Sprintf("failed to put in vc store: %s", err.Error())
+			inputs[0].Call("respondWith", js.Global().Get("Promise").Call("resolve", m))
```

Comment:
tight coupling between wasm and vue.js? might not matter if this is just a "demo" edge-agent